servers/cu/src/domain/lib/chainEvaluation.js

```diff
@@ -21,9 +21,7 @@ function loadLatestEvaluationWith ({ findEvaluation, findLatestProcessMemory, lo
 
     return findEvaluation({
       processId: ctx.id,
-      to: ctx.to,
-      ordinate: ctx.ordinate,
-      cron: ctx.cron
+      messageId: ctx.messageId
```

Comment:
There are cases where `messageId` will be undefined, like the case of requests to `/cron` and `/dry-run`. 

Perhaps this should attempt to call `findEvaluation` **iff** `messageId` is provided?